extensions/amp-subscriptions/amp-subscriptions.md

```diff
@@ -262,97 +483,215 @@ The vendor service configuration must reference the service ID and can contain a
 
 See the vendor service's documentation for details.
 
-## Authorization endpoint and entitlements
+#### Available vendor services
 
-Authorization is an endpoint provided by the local service and called by the AMP Runtime. It is a credentialed CORS GET endpoint. This endpoint returns the Entitlements object that can be used by the Content Markup to hide or show different parts of the document. Authorization endpoint is specified using the "authorizationUrl" property in the config.
+- [amp-subscriptions-google](../amp-subscriptions-google/amp-subscriptions-google.md)
 
-The Entitlement response returned by the authorization endpoint must conform to the predefined format:
+### Service score factors
 
+If no service returns an entitlement that grants access, all services are compared by calculating a score for each and the highest scoring service is selected. Each service has a `"baseScore"` (default 0). A value < 100 in the `baseScore` key in any service configuration represents the initial score for that service. If no `baseScore` is specified it defaults to `0`.
```

Comment:
What is meant by "service"?

Reply:
The service refers to either a local service or one of the vendor services configured in the component. Do you think we should call that out in this section or perhaps in a dedicated section somewhere?

Reply:
Got it. I'll take another look at it when some of the other details above are filled in. I think it ight be okay. The sentence might just need to get broken up :)